samples/charts/data-chart/scatter-single-bubble-series/App.razor

```diff
@@ -0,0 +1,76 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
```

Comment:
rename sample folder to: "scatter-bubble-chart-single-source"

Reply:
resolved, pending